RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate REST API - getRoomInfo to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce
Open a chat:

* check if the messages are loading;
* check if the messages are updated;
* check the threads;
* check the rooms info;
* Check if the list of chats is still working;

## Screenshots

## Types of changes




## Checklist



## Further comments
